### ISSUE TITLE
Add "headers" to middleware

### ISSUE THREAD

**Issue by GiordanoArman** [fiznool/express-mongo-sanitize#5]
What about adding the headers to the array of req values that get sanitized? Let's say an application uses an Authorization header, which or example could be "email@address.tk:thisIsThePassWord", right now this module would not protect against an attack that exploits the headers feature.

The bit to change would be:

```javascript
function middleware(options) {
  return function(req, res, next) {
    ['body', 'params', 'query', 'headers'].forEach(function(k) {
      if(req[k]) {
        req[k] = sanitize(req[k], options);
      }
    });
    next();
  };
}
```

Let me know what you think!

**Comment by fiznool** [comment]
This is a good idea. If you could put together a PR with corresponding tests I would gladly merge this in!

**Comment by fiznool** [comment]
Fixed in v2.0.0.
